src/components/Home/CareerPart2.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import Data from "./JobsData.json";
+import "../styles/careersection.css";
+
+const CareerPart2 = () => {
+    return(
+        <div className="div">
+            <h1 className="header">current open positions</h1>

```

Comment:
You gonna be using this component in one page and as a convention, each page should have only 1 <h1> tag, so you better change this tag to any heading <h2...6> but not <h1>